format.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const format: (random: (size: number) => string[], alphabet: string, size: number) => string;
+export = format;
```

Comment:
All modules here has only one export so you may don't use tricky funky synthetic exports.